### PR TITLE
Add SuppressedExternally to SuppressionStates enum. Release 1.5.25 as non-beta

### PR DESCRIPTION
**Pull request by michaelcfanning**
Unit tests already exist for this enum. @lgolding @rtaket

### CODE REVIEW

File: src/Sarif/CodeGenHints.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
           "Assignment",
           "Branch",
           "Call",
-          "CadsdalReturn",
```
Comment (lgolding) [inline src/Sarif/CodeGenHints.json]: eek

File: src/Sarif/CodeGenHints.json [hunk 2]
```diff
@@ -59,16 +59,6 @@
...
-  "AnnotatedCodeLocation.Kind": [
-    {
-      "kind": "EnumHint",
-      "arguments": {
-        "typeName": "AnnotatedCodeLocationKind",
-        "flags": true,
-        "zeroValueName": "None"
-      }
-    }
-  ],
```
Comment (lgolding) [inline src/Sarif/CodeGenHints.json]: Ah, there were two of these, good catch.

**Comment by lgolding** [comment]
:+1:

File: src/Sarif/CodeGenHints.json (hunk 1 above)
Comment (michaelcfanning) [inline src/Sarif/CodeGenHints.json]: bad news, can you believe it?

**Pull request merged**